app/spell_check_test.py

```diff
@@ -37,5 +37,13 @@ def test_sort_by_count(self):
         self.assertEqual(sort_by_count([['test', 10], ['text', 12], ['other', 8], ['thing', 40]]), [['thing', 40], ['text', 12], ['test', 10], ['other', 8]])
         self.assertEqual(sort_by_count([['test', 10], ['text', 10], ['other', 10], ['thing', 10]]), [['test', 10], ['text', 10], ['other', 10], ['thing', 10]])
 
+    # def test_check_other_langs_irish(self):
```

Comment:
Why is this test commented out?

Reply:
These tests were built using the old load_dict file/function and I havent quite gotten them fully fixed, was planning on doing that in a later commit so we could get this one in